### ISSUE TITLE
Wallaby never starts up with latest version of VSCode

### ISSUE THREAD

**Issue by deshiknaves** [wallabyjs/public#3408]
### Issue description or question

Wallaby never starts on latest version of VSCode with vite. This was working until now. I can't get a diagnostic report because it never starts up. There is no output from wallaby. I've tried uninstalling and installing. Here is some output from the extension host:

```shell
2024-10-22 15:57:10.729 [error] TypeError: Cannot read properties of undefined (reading 'foreground')
    at u (/Users/user/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.388/dist/index.js:15:162858)
    at /Users/user/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.388/dist/index.js:15:163212
    at Array.forEach (<anonymous>)
    at p (/Users/user/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.388/dist/index.js:15:162944)
    at a.loadTheme (/Users/user/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.388/dist/index.js:15:163587)
    at U._setPreferences (/Users/user/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.388/dist/index.js:15:137607)
    at r.<anonymous> (/Users/user/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.388/dist/index.js:15:94416)
    at r.emit (node:events:519:28)
    at r.emit (node:domain:488:12)
    at r._started (/Users/user/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.388/wallaby85e603/client.js:14:10953)
    at WebSocket.<anonymous> (/Users/user/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.388/wallaby85e603/client.js:14:5518)
    at WebSocket.emit (node:events:519:28)
    at WebSocket.emit (node:domain:488:12)
    at Receiver.receiverOnMessage (/Users/user/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.388/wallaby85e603/node_modules/ws/lib/websocket.js:1209:20)
    at Receiver.emit (node:events:519:28)
    at Receiver.emit (node:domain:488:12)
    at /Users/user/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.388/wallaby85e603/node_modules/ws/lib/receiver.js:608:16
    at node:internal/process/task_queues:140:7
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at AsyncResource.runMicrotask (node:internal/process/task_queues:137:8)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by smcenlly** [comment]
Thanks for reporting the problem.

We've released a new version of Wallaby for VS Code that should fix your problem, `v1.0.389`.

Could you also please let us know:
* What theme you are using in VS Code? 
* Are you using VS Code remote development?

**Comment by deshiknaves** [comment]
Yep, this is all working now. I'm using CodeSandboxTheme. I do use remote development from time to time, but this was on a non remote session.

Name: CodeSandbox Theme
Id: CodeSandbox-io.codesandbox-projects-theme
Description: A CodeSandbox theme for Visual Studio Code
Version: 1.1.0
Publisher: CodeSandbox
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=CodeSandbox-io.codesandbox-projects-theme
